stratum/hal/bin/barefoot/BUILD

```diff
@@ -208,3 +211,63 @@ pkg_deb(
     prerm = "deb/prerm",
     version = "0.0.1",
 )
+
+genrule(
+    name = "gen_sde_version",
+    outs = ["bf_sde_version.txt"],
+    cmd = "printf $(sde_ver) > $@",
+)
+
+download_pkgs(
+    name = "stratum_bfrt_pkg_download",
```

Comment:
Change name.
This base image is not limited to bfrt, but applies to bf as well.